ic-agent/src/agent/agent_test.rs

```diff
@@ -85,10 +85,11 @@ async fn query_error() -> Result<(), AgentError> {
 #[cfg_attr(not(target_family = "wasm"), tokio::test)]
 #[cfg_attr(target_family = "wasm", wasm_bindgen_test)]
 async fn query_rejected() -> Result<(), AgentError> {
```

Comment:
@ericswanson-dfinity Here is the integration test for mapping to replica errors on a rejected response.

Reply:
Right, but I notice that this has `None` for the error_code.  Will error codes be available for queries?

Reply:
Yes. I extended the test now to also include `Some("Error code")` in the error_code field.